modulemd/v2/modulemd-module-index.c

```diff
@@ -353,9 +353,9 @@ compare_stream_SVC (gconstpointer a, gconstpointer b)
 {
   ModulemdModuleStream *a_ = *(ModulemdModuleStream **)a;
   ModulemdModuleStream *b_ = *(ModulemdModuleStream **)b;
-  g_autofree gchar *a_nsvc = modulemd_module_stream_get_nsvc_as_string (a_);
-  g_autofree gchar *b_nsvc = modulemd_module_stream_get_nsvc_as_string (b_);
-  return g_strcmp0 (a_nsvc, b_nsvc);
+  g_autofree gchar *a_nsvca = modulemd_module_stream_get_NSVCA_as_string (a_);
+  g_autofree gchar *b_nsvca = modulemd_module_stream_get_NSVCA_as_string (b_);
+  return g_strcmp0 (a_nsvca, b_nsvca);
```

Comment:
Is it a good idea to do it this way? Shouldn't we have this deprecated for an explicitly SVCA variant?

Reply:
This is a static, internal function. I can rename it to `compare_stream_SVCA()` if you prefer. It's called in exactly one place, as a comparison function for a sort.

Reply:
I'd prefer us to be explicit, even with the function definition for an internal function. :)

Reply:
Fixed and pushed.